design/README.md

```diff
@@ -1,6 +1,9 @@
 # Design
 
+* [No Silver Bullet (1986)](http://faculty.salisbury.edu/~xswang/Research/Papers/SERelated/no-silver-bullet.pdf)
+    > Most of the big past gains in software productivity have come from removing artificial barriers that have made the accidental tasks inordinately hard, such as severe hardware constraints, awkward programming languages, lack of machine time. How much of what software engineers now do is still devoted to accidental, as opposed to essential?
+
 * [Traits: A Mechanism for Fine-Grained Reuse](http://scg.unibe.ch/archive/papers/Duca06bTOPLASTraits.pdf)
 
-* :scroll: [Out of the Tar Pit](out-of-the-tar-pit.pdf)
-    > We identify common causes of complexity and discuss general approaches which can be taken to eliminate them where they are accidental in nature.
\ No newline at end of file
+* [Out of the Tar Pit (2006)](out-of-the-tar-pit.pdf)
```

Comment:
We want to keep the :scroll: on this, as we have the paper hosted within the directory.